preload/data/ui/chart-editor/dialogs/user-guide.xml

```diff
@@ -18,6 +18,8 @@
 					<item action="Select Note/Event" keybind="Left Click on Note" />
 					<item action="Select Multiple Notes/Events" keybind="Ctrl + Left Click" />
 					<item action="Remove Note/Event" keybind="Right Click on Note" />
```

Comment:
```suggestion
					<item action="Remove Note/Event" keybind="Right Click on Note" />
					<item action="Show Note Options" keybind="Shift + Right Click on Note" />
```

Reply:
Oh right, never noticed this shortcut wasn't mentioned. Thanks!